src/main/java/org/openrewrite/staticanalysis/EqualsAvoidsNull.java

```diff
@@ -171,14 +200,13 @@ private J.Binary literalsFirstInComparisonsNull(J.MethodInvocation m,
 
                     private J.MethodInvocation literalsFirstInComparisons(J.MethodInvocation m,
                                                                           Expression firstArgument) {
-                        if (!(firstArgument instanceof J.Literal) && !(m.getSelect() instanceof J.Literal)) {
```

Comment:
Try to provide the conditions with proper inline comments or extract them into cleanly named methods. In my opinion, this is better than using overhead in double `if` statements that are technically the same. There’s no need to split them explicitly.

In this case, it’s the opposite, but kind of the same as above. It’s always the same, so we want to communicate this right away. In this case, the indentation and additional `if` are, in my eyes, boilerplate and overhead as well.